src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,81 @@
+.cards__card {
+  &:nth-child(n) {
+    margin-bottom: 46px;
+  }
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+}
+
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  width: get-width(200px, 4, 48px);
+  margin: 0 auto;
+  padding: 50px 0;
+
+  .card {
+    display: flex;
+    flex-direction: column;
+    width: 200px;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    background: $color-white;
+    padding: 32px 16px 16px;
+    
+    &__image {
+      width: 160px;
+      height: 134px;
+      margin: 0 auto 40px;
+    }
+
+    &__title {
+      @include text($font-size-s, 500, $color-main-text, 18px);
+      margin-bottom: 4px;
+    }
+
+    &__code {
+      @include text($font-size-xs, 400, $color-secondry-text, 14px);
+      margin-bottom: 16px;
+    }
+
+    &__reviews {
+      @include text($font-size-xs, 400, $color-secondry-text, 14px);
+
+    }
+    
+    &__price {
+      display: flex;
+      justify-content: space-between;
+      margin-bottom: 16px;
+
+      &-text {
+        @include text($font-size-s, 400, $color-secondry-text, 18px);
+      }
```

Comment:
don't do it. make it an `&__price-text`. An element can't contain another element according BEM